build/node-compose.yml

```diff
@@ -29,11 +27,12 @@ services:
       CARTESI_BLOCKCHAIN_IS_LEGACY: "false"
       CARTESI_BLOCKCHAIN_GENESIS_BLOCK: "1"
       CARTESI_BLOCKCHAIN_FINALITY_OFFSET: "1"
-      CARTESI_CONTRACTS_DAPP_ADDRESS: "0x70ac08179605AF2D9e75782b8DEcDD3c22aA4D0C"
+      CARTESI_BLOCKCHAIN_READ_DEPTH: "1"
```

Comment:
This variable doesn't exist; see https://github.com/cartesi/rollups-node/blob/main/docs/config.md

Reply:
I don't recall where it comes from. Removed at 9e6a40852196dad533525be4c928c4b542c650cb